core/src/test/scala/akka/persistence/r2dbc/snapshot/R2dbcSnapshotStoreSpec.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.r2dbc.snapshot
+
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.adapter._
+import akka.persistence.CapabilityFlag
+import akka.persistence.r2dbc.{ TestConfig, TestDbLifecycle }
+import akka.persistence.snapshot.SnapshotStoreSpec
+
+class R2dbcSnapshotStoreSpec extends SnapshotStoreSpec(TestConfig.config) with TestDbLifecycle {
+  def typedSystem: ActorSystem[_] = system.toTyped
+
+  protected override def supportsMetadata: CapabilityFlag = true
```

Comment:
do we need two tests, one with true and one with false?

Reply:
Enabling just adds one more test case to the existing so I think that this is fine?